app.test.js

```diff
@@ -134,3 +135,57 @@ test('should respond with a 400 if the provided id is not valid (eg not a number
 })
 })
 
+describe('GET /api/articles/:article_id/comments',()=>{
+    test('Should respond with 200 and the correct comments for the article',()=>{
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body})=>{
+            const { comments } = body;
+            expect(comments).toBeSorted({"key": "created_at", "ascending": true})
+            for(let i = 0; i < body.length; i++){
+                expect(body.comment),toHaveProperty('comment_id')
```

Comment:
you seem to have a comma here on line 147 instead of a . where you are trying to use the .toHaveProperty jest matcher

Reply:
Wow, well spotted. Thanks!